### DISCUSSION TITLE
Nomination of @J-N-K

### DISCUSSION THREAD

**Discussion by cweitkamp** [openhab/openhab-addons#14695] [Add-on Maintainers Discussions]
I would like to nominate @J-N-K as new maintainer. He was quite active in the [last couple of weeks](https://github.com/openhab/openhab2-addons/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3AJ-N-K+) and he convinced me with his developer skills, his interest in the new build system and knowledge about the frameworks features to implement bindings (see previous work on [ESH](https://github.com/eclipse/smarthome/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3AJ-N-K)). He is having fun to support the community and likes to improve openHAB. I already got in touch with him and he is willing to join the team.

**Comment by cweitkamp** [comment]
@openhab/2-x-add-ons-maintainers You may want to add your vote / voice / :+1: here to address https://github.com/openhab/openhab2-addons/pull/5160#issuecomment-475009853.

**Comment by davidgraeff** [comment]
@wborn

**Comment by davidgraeff** [comment]
@kaikreuzer Everyone has voted now

**Comment by kaikreuzer** [comment]
As @cweitkamp did the nomination, I'd leave it to him to add @J-N-K to the team (note: you should not need me for that!)

**Comment by davidgraeff** [comment]
Actually you are the only one who can do that (you have the badge "maintainer" in the team list)

**Comment by cweitkamp** [comment]
@kaikreuzer I tried it but only can find a "Leave team" button in the "Members" section.

**Comment by kaikreuzer** [comment]
Ok, it seems that the team maintainer role was missing - you should now also see an "add a member" button.
But I have now added @J-N-K as a team member already, welcome! 👏

**Comment by cweitkamp** [comment]
Yes, confirmed. I can see it now. Thanks.

**Comment by J-N-K** [comment]
Thank you very much. I just finished a first review.
